code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -599,7 +599,7 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/M, alien)
 		for(var/datum/internal_organ/I in H.internal_organs)
 			if(I.damage)
```

Comment:
`if(I.damage && I.damage <= 5 && M.bodytemperature <= 169)`, and remove the two lines below.
In the current state it'll return whenever an organ has more than 5 damage, instead of skipping over.